java/code/src/com/redhat/rhn/taskomatic/TaskomaticApi.java

```diff
@@ -221,6 +221,20 @@ public Date scheduleSingleSatBunch(User user, String bunchName,
         return (Date) invoke("tasko.scheduleSingleSatBunchRun", bunchName, params);
     }
 
+    /**
+     * Creates a new single gatherer schedule
+     * @param user shall be org admin
+     * @param bunchName bunch name
+     * @param params parameters for the bunch
+     * @return date of the first schedule
+     * @throws TaskomaticApiException if there was an error
+     */
+    public Date scheduleGathererRefresh(User user, String bunchName,
```

Comment:
With this name, I think we should limit the use and hardcode the `bunchName`.
Otherwise it can be used to schedule any bunch which requires `Sat`elite permissions using this new endpoint.

See also scheduleRepoSync() .....

Reply:
the activation key thing is part of the another pr i made. but i didn't make a separate branch. but i managed to get them removed.

Reply:
I think @mcalmer wants you to get rid of the `bunchName` parameter and pass the `gatherer-matcher-bunch` on line 235.

Moreover, I'd suggest passing a `vhmLabel` parameter and crafting the `params` map inside of this method (instead of the whole `params` map as a param).

It'd be also nice to do some additional permission checking here (just call `VirtualHostManagerFactory.lookupByLabelAndOrg` based on given label and `user.getOrg`. When the result is null -> throw an exception (`IllegalStateException` probably)).